src/linear_algebra/eigenspace.lean

```diff
@@ -159,21 +159,14 @@ end minpoly
 lemma exists_eigenvalue [is_alg_closed K] [finite_dimensional K V] [nontrivial V] (f : End K V) :
   ∃ (c : K), f.has_eigenvalue c :=
 begin
-  classical,
-  -- Choose a nonzero vector `v`.
-  obtain ⟨v, hv⟩ : ∃ v : V, v ≠ 0 := exists_ne (0 : V),
-  -- The infinitely many vectors v, f v, f (f v), ... cannot be linearly independent
-  -- because the vector space is finite dimensional.
-  have h_lin_dep : ¬ linear_independent K (λ n : ℕ, (f ^ n) v),
-  { apply not_linear_independent_of_infinite, },
```

Comment:
Was this just not used!?

Reply:
The problem is that it was a mishmash of an old proof (the one from Axler, which looks at `(f^n) v` and says they must be linearly dependent, and a newer (higher-tech) proof, that just says some polynomial on `f` itself vanishes. Whoever switched it over to the newer proof didn't do it completely: they still used `v` to show that `aeval f p` is not a unit. This cleanup just uses the fact that `aeval f p = 0`.